### PR TITLE
Update ReadME.md (add Features/Gallery, etc.)

### PR DESCRIPTION
**Pull request by samuelroman74**
Added features list and gallery, fixed some grammatical errors, clarified itch.io installation technique, and further compartmentalized how people could contribute to the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +60,7 @@ Installing
 Wesnoth can be played on Windows, macOS, and Linux, and can be installed using:
 * [Steam](https://store.steampowered.com/app/599390/Battle_for_Wesnoth/) (all)
 * [itch.io](https://wesnoth.itch.io/battle-for-wesnoth) (Windows and macOS)
+    * Download folder, unzip, then run/open wesnoth.exe
```
Comment (Pentarctagon) [inline README.md:63]: This is accurate for the standalone download for Windows, however it's incorrect for macOS as well as when played through the itch.io client.

File: README.md [hunk 2]
```diff
@@ -56,7 +88,7 @@ For any questions, the best place to start is to simply join Wesnoth's forums, D
 Donations
 ---------
 
-Donations can be sent via [Liberapay](https://liberapay.com/Wesnoth) as well as when downloading through itch.io. All money donated is used to pay for project expenses such as server rent and art commissions, and is very much appreciated.
+Donations can be sent via [Liberapay](https://liberapay.com/Wesnoth) as well as when downloading through itch.io. All money donated is used to pay for project expenses such as server rent and art commissions and is very much appreciated.
```
Comment (Pentarctagon) [inline README.md:91]: There's https://www.spi-inc.org/projects/wesnoth/ as well now, actually.

File: README.md [hunk 3]
```diff
@@ -8,7 +8,32 @@
 About
 =====
 
-**The Battle for Wesnoth** is an Open Source, turn-based tactical strategy game with a high fantasy theme, featuring both singleplayer and online/hotseat multiplayer combat. Fight a desperate battle to reclaim the throne of Wesnoth, or take hand in any number of other adventures.
+**The Battle for Wesnoth** is an Open Source, turn-based tactical strategy game with a high fantasy theme, featuring both singleplayer and online/hotseat multiplayer combat. Fight a desperate battle to reclaim the throne of Wesnoth, search for the key to immortality, or take part in any number of other adventures.
```
Comment (Pentarctagon) [inline README.md:11]: singleplayer is two works.

File: README.md [hunk 4]
```diff
@@ -8,7 +8,32 @@
...
+-------
+
+![Screenshot_20230213_014035](https://user-images.githubusercontent.com/87458186/218564775-51fd00e4-417e-476e-83b3-eeaa7c538ce8.png)
+![Screenshot_20230213_014257](https://user-images.githubusercontent.com/87458186/218564861-a7361c7d-081c-49e4-a4e7-bf3302525986.png)
+![Screenshot_20230213_020340](https://user-images.githubusercontent.com/87458186/218564900-da341ba3-93f7-4491-8bfc-475776518ca2.png)
+
+Features
+--------
+
+* 17 different campaigns, each with their own tactical challenges, emotionally charged story, and multiple difficulty levels.
```
Comment (cooljeanius) [inline README.md:23]: Is the "17" number currently accurate? It keeps changing with every release, and that's just the mainline campaigns... (also there's the question of whether to count stuff like the Tutorial, or multiplayer campaigns like WC...)

File: README.md [hunk 5]
```diff
@@ -8,7 +8,32 @@
...
+Features
+--------
+
+* 17 different campaigns, each with their own tactical challenges, emotionally charged story, and multiple difficulty levels. 
+* Detailed world and storytelling with hundreds of sprites, animations, and characters.
+* Different skills/traits, even within the same troop class.
+* Villages to conquer, keeps to recruit units with, and a variety of other innovative tactical features that can give you the edge in combat. 
+* Day/Night cycle that affects how races play, feel, and attack.
+* Enemies that adapt to your moves and aim for the most strategic course of action.
+* Map editor with multiple tools, 200+ types of terrain, and the ability for you to save/share maps with others. Create your own scenarios/battles for near infinite     replayability.
```
Comment (cooljeanius) [inline README.md:29]: excess spaces here
```suggestion
* Map editor with multiple tools, 200+ types of terrain, and the ability for you to save/share maps with others. Create your own scenarios/battles for near infinite replayability.
```

File: README.md [hunk 6]
```diff
@@ -8,7 +8,32 @@
...
+* Online/multiplayer gameplay to play with friends and strangers.
+* Local and AI gameplay with dozens of maps and customizable starting conditions and settings for the ultimate tactical experience. 
+* Changeable settings for base gameplay, with nearly limitless options for different hotkey configurations.
+* Tons of user-made add-ons, including new map packs, eras, and entire campaigns.
+* Guides that update as you play through the game, filled with tactical tips, refreshers on gameplay, unit/race types, and information on the rich world that permeates   the Battle of Wesnoth.
```
Comment (cooljeanius) [inline README.md:34]: excess spaces here, too
```suggestion
* Guides that update as you play through the game, filled with tactical tips, refreshers on gameplay, unit/race types, and information on the rich world that permeates the Battle of Wesnoth.
```

**Review by cooljeanius — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (CelticMinstrel) [reply README.md:11]: I'd expect it to be hyphenated in this context.

File: README.md (hunk 4 above)
Comment (knyghtmare) [reply README.md:23]: For now, I guess. Might shrink or increment. Who knows

**Comment by cooljeanius** [comment]
@samuelroman74 do you intend to continue with this PR?

**Pull request closed without merging**